ui/src/svelteStore.test.ts

```diff
@@ -12,13 +12,15 @@ import * as svelteStore from "./svelteStore";
 describe("waitUntil", () => {
   test("current value matches", async () => {
     const store = svelteStore.writable(true);
-    const value = await svelteStore.waitUntil(store, x => x);
-    expect(value).toBe(true);
+    const value = await svelteStore.waitUntil(store, x => x.toString());
```

Comment:
Just for my own curiosity, why is this needed now?

Reply:
The change is not strictly necessary. The tests would have passed without it. That’s the reason why I changed it: to have a test that only passes with the new implementation. Calling `.toString()` I just a quick and arbitrary way of having some function that transforms the input.